src/ralph/backends/database/base.py

```diff
@@ -66,6 +70,32 @@ class StatementParameters:
     search_after: Optional[str] = None
     pit_id: Optional[str] = None
 
+    def __post_init__(self):
+        # Check that both `homePage` and `name` are provided if `account is being used`
+        if (self.agent__account__name is not None) != (
+            self.agent__account__home_page is not None
+        ):
+            raise BackendParameterException(
+                "Invalid agent parameters: home_page and name are both required"
+            )
```

Comment:
Isn't it already enforced by the `AccountActorField` model, which does not accept any optionnal?

Reply:
I believe it is but the `AccountActorField` is used at the API level. Since this is at the database level, I thought it could be useful to add this extra security, in case this part of the code is used in another context.

 Do you think I should remove it ?